app/src/main/java/com/twelvesixty/peak/UserProfileActivty.java

```diff
@@ -0,0 +1,87 @@
+package com.twelvesixty.peak;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.text.Layout;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+public class UserProfileActivty extends AppCompatActivity {
+    LinearLayout defaultProfileLayout;
+    LinearLayout editableLayout;
+    Button editProfileButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_user_profile_activty);
+
+        // find views by id
+        editableLayout = findViewById(R.id.linearLayout_editable);
+        defaultProfileLayout = findViewById(R.id.linearLayout_default);
+        editProfileButton = findViewById(R.id.button_EditProfile);
+
+        // set editable layout view to GONE
+        editableLayout.setVisibility(View.GONE);
+
+    }
+
+    // show users content from profile or provide editable content
+    public void onEditProfileButtonClick(View view) {
+
+        String buttonText = editProfileButton.getText().toString().toLowerCase();
+
+        if(buttonText.equals("edit profile")) {
+
+            flipToEditMode();
+            buttonText = "Save";
+            editProfileButton.setText(buttonText);
+
+        } else {
+
+            buttonText = "Edit Profile";
+            editProfileButton.setText(buttonText);
+
+            EditText bioFormEditView = findViewById(R.id.textView_Bio_Editable);
+            String bioFormInput = bioFormEditView.getText().toString();
+
+            EditText favoriteResortEditView = findViewById(R.id.textView_favoriteResort_Editable);
+            String favoriteResortFormInput = favoriteResortEditView.getText().toString();
+
```

Comment:
Editing the favorite resort from the profile could get messy. Might just want to remove this part